test/app/app.spec.js

```diff
@@ -1,56 +1,58 @@
-"use strict";
+'use strict';
 
 let DemoAppTester = require('./lib/demo-app-tester');
-let webdriver = require('selenium-webdriver');
-let expect = require('chai').expect;
 let path = require('path');
+let JuttleMoment = require('juttle/lib/moment').JuttleMoment;
 
-const TEST_TIMEOUT = 10000;
-const BROWSER = "from";
-const WEBDRIVER_URL = "http://localhost:9515/";
+const TEST_TIMEOUT = 30000;
 
+function toJuttleMoment(value) {
+    return new JuttleMoment({ raw: value}).valueOf();
+}
 
-// skipped while webdriver is not properly configured in travis
-describe.skip("demo-app", function() {
+describe('demo-app', function() {
     this.timeout(TEST_TIMEOUT);
-    let driver;
     let demoAppTester;
-    beforeEach(() => {
-        driver = new webdriver.Builder()
-            .forBrowser(BROWSER)
-            .usingServer(WEBDRIVER_URL)
-            .build();
-        demoAppTester = new DemoAppTester(driver);
+
+    before(function(done) {
+        demoAppTester = new DemoAppTester();
+        demoAppTester.start(done);
     });
 
-    it("open juttle program with no inputs", () => {
-        return demoAppTester.loadFile(path.join(__dirname, "juttle", "no-inputs.juttle"))
-           .then(() => {
-               return demoAppTester.getLoggerOutput('myLogger');
-           })
-           .then((value) => {
-               expect(JSON.parse(value)).to.deep.equal([{time: new Date(0).toISOString(), v: 10}]);
-           });
+    after(() => {
+        demoAppTester.stop();
     });
 
-    it("open juttle program with an input, fill it out, and run", () => {
-        return demoAppTester.loadFile(path.join(__dirname, "juttle", "one-input.juttle"))
-            .then(() => {
-                return demoAppTester.findInputControl("a");
-            })
-            .then((inputElem) => {
-                inputElem.sendKeys("AAA");
-            })
-            .then(demoAppTester.clickPlay)
-            .then(() => {
-                return demoAppTester.getLoggerOutput('myLogger');
-            })
-            .then((value) => {
-                expect(JSON.parse(value)).to.deep.equal([{time: new Date(0).toISOString(), v: "AAA"}]);
-            });
+    it('open juttle program with no inputs', () => {
+        return demoAppTester.run({
+            path: path.join(__dirname, 'juttle', 'no-inputs.juttle')
+        })
+        .then(() => {
+            return demoAppTester.waitForTextOutputToContain('output',[
+                { time: toJuttleMoment(0), value: 10 },
```

Comment:
Instead of using JuttleMoment, you can use `new Date(0).toISOString()`. In general we are thinking of making `JuttleMoment` more private than public.